TGC/decisions/Germany VS Hungary Desc.txt

```diff
@@ -0,0 +1,63 @@
+political_decisions = {
+
+##Treaty of Vienna##
+
+treaty_of_Vienna_hungary = {
+		picture = treaty_signing
+		potential = {
+			
+			tag = GER
+                        has_global_flag = greatest_germany

```

Comment:
If this is an extension of GEF (judging by the use of this flag) please also have NOT = { has_global_flag = extra_flavour_disabled } in the potential conditions since GEF is currently opt-in via the setup decisions.

Reply:
threw out the GEF modifiers